pages/dkp/kommander/1.3/release-notes/1.3.1/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.3.1
+title: Release Notes Kommander 1.3.1
+menuWeight: 0
+excerpt: View release-specific information for Kommander 1.3.1
+enterprise: false
+---
+
+**D2iQ&reg; Kommander&reg; version 1.3.1 was released on 11, March 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# Breaking changes
+
+In Kommander 1.3, unless a default NetworkPolicy existed prior to upgrading to 1.3, a default NetworkPolicy was created that allowed traffic originating only within your Projects namespace. Pods could not talk to Pods outside of that namespace. In some cases this is not a desired default state. Now no default NetworkPolicy is created and no blocking is done. 
```

Comment:
I got a little confused when reading this from the vantage point of somebody who might have started using Kommander with 1.3, and might not have been aware of this policy, or might have desired this policy.  With that in mind, I suggest this:

```suggestion
In Kommander 1.3, all new projects were created with a default NetworkPolicy that only allowed traffic originating within your Projects namespace.  This policy was also created in all existing projects that did not already have a NetworkPolicy when upgrading to Kommander 1.3.

The result of this default policy was that Pods could not talk to Pods outside of that namespace. In some cases this was not a desired default state.  As of Kommander 1.3.1, no default NetworkPolicy is created in new (or existing projects) and thus no traffic blocking is done by default.  
```